source/changelog.txt

```diff
@@ -12,6 +12,32 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v1.7.1
+------
+
+*Released February 16, 2023*
+
+- :issue:`MONGOSH-1087` An earlier enhancement to autocomplete that is
+  available now.
+- :issue:`MONGOSH-1378` Fixes connectivity issues when ``mongosh`` is
+  installed using Homebrew.
+
+`Full release notes available on JIRA
+<https://jira.mongodb.org/secure/ReleaseNote.jspa?projectId=17381&version=35548>`__.
+
+v1.7.0
+------
+
+*Released February 10, 2023*
+
+- :issue:`MONGOSH-57` Display a warning when connecting to impostor
+  MongoDB instances such as CosmosDB or DocumentDB.
```

Comment:
I’d check with @mmarcon about whether to include the names of these products here.